src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -39,22 +39,23 @@ export default function CSVFileImport({url, title}: CSVFileImportProps) {
         url,
         params: {
           csvFileName: encodeURIComponent(file.name)
-        }
-      })
+        },
+        headers: {
+          Authorization: `Basic ${localStorage.getItem("auth_token")}`,
```

Comment:
Client should get **authorization_token** value from browser localStorage

Reply:
Auth is the same as authorization_token, but shorter